python/docs/source/getting_started/quickstart_df.ipynb

```diff
@@ -218,27 +218,23 @@
    "metadata": {},
    "outputs": [
     {
-     "data": {
-      "text/html": [
-       "<table border='1'>\n",
-       "<tr><th>a</th><th>b</th><th>c</th><th>d</th><th>e</th></tr>\n",
-       "<tr><td>1</td><td>2.0</td><td>string1</td><td>2000-01-01</td><td>2000-01-01 12:00:00</td></tr>\n",
-       "<tr><td>2</td><td>3.0</td><td>string2</td><td>2000-02-01</td><td>2000-01-02 12:00:00</td></tr>\n",
-       "<tr><td>3</td><td>4.0</td><td>string3</td><td>2000-03-01</td><td>2000-01-03 12:00:00</td></tr>\n",
-       "</table>\n"
-      ],
-      "text/plain": [
-       "DataFrame[a: bigint, b: double, c: string, d: date, e: timestamp]"
-      ]
-     },
-     "execution_count": 8,
-     "metadata": {},
-     "output_type": "execute_result"
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
```

Comment:
Let's present the output results in `text` format instead of `text/html` format to avoid formatting errors.

Reply:
Hm, this example should show the output nicely as `spark.sql.repl.eagerEval.enabled` is enabled. Wonder if we can fix the docs instead.

Reply:
Dark theme:
<img width="802" alt="image" src="https://github.com/apache/spark/assets/15246973/88151590-ea01-4bbc-8678-5bca2a20dc74">

Light theme:
<img width="811" alt="image" src="https://github.com/apache/spark/assets/15246973/3505f951-05ba-4620-ab65-779acfed4828">